### ISSUE TITLE
Is it possible to open source the test set?

### ISSUE THREAD

**Issue by Henglin-Liu** [multimodal-reasoning-lab/Bagel-Zebra-CoT#13]
Thank you for your excellent work. In the link https://huggingface.co/datasets/multimodal-reasoning-lab/Zebra-CoT, it seems that there is only the training set. May I ask if the corresponding test set can be open-sourced for the convenience of testing? It is convenient to improve the original method so as to expand the influence of the work.
